wgpu/utils/imgui/imgui_renderer.py

```diff
@@ -166,6 +164,12 @@ def render(self):
                 }
             ],
         )
+
+        imgui.new_frame()
+        self._update_gui_function()
+        imgui.render()
```

Comment:
Do you think this would be worthwhile? To guarantee we never leave imgui in a pending state?

```suggestion
        imgui.new_frame()
        try:
            self._update_gui_function()
        finally:
            imgui.render()
```

Reply:
Yeah, you could handle the exception and ignore it if you like, and proceed with iterating